docs-ref-conceptual/manage-cloud-sets-azure-cli.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Managing Cloud Sets with Azure CLI 2.0
+description: Create, log in, and manage different cloud sets with the Azure CLI 2.0.

```

Comment:
I would try to get in some reference to "government" and "region" in the description, as one of the drivers for this is helping people log into US Government cloud or China/Germany regional clouds.